go.mod

```diff
@@ -0,0 +1,121 @@
+module github.com/osbuild/image-builder-cli
+
+go 1.23
```

Comment:
Should we match the version of osbuild/images? I like to use always the greatest and latest, but I would like to prevent any future issues with distros having too old versions.

Reply:
I have no strong opinion here (and flip-flopped a bit when adding this), I'm happy to move to the version from images. I also hope that we will distribute this (at least optionally) similar to bib in a container so we have more freedom) - but for now let's be convervative and move to the images version